pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -277,6 +283,33 @@ func manageOpenShiftAPIServerImageImportCA_v311_00_to_latest(openshiftConfigClie
 	}
 }
 
+func ensureOpenShiftAPIServerTrustedCA_v311_00_to_latest(client coreclientv1.CoreV1Interface, recorder events.Recorder) error {
+	const injectCAAnnotationKey = "config.openshift.io/inject-trusted-cabundle"
```

Comment:
no more consts! :-)

Reply:
Consts are helpful if you have clumsy fingers like I do :) But alright, this one will appear only in one spot in this fucntion anyway